ebt/Application.vala

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright 2021 elementary, Inc. (https://elementary.io)
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program; if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA
+ *
+ * Authored by: Torikulhabib <torik.habib@gamail.com>
+ *
+ */
+
+public class BluetoothApp : Gtk.Application {
+    public const OptionEntry[] OPTIONS_BLUETOOTH = {
+        { "silent", 's', 0, OptionArg.NONE, out silent, "Run the Application in background", null},
+        { "send", 'f', 0, OptionArg.NONE, out send, "Open file to send via bluetooth", null },
+        { "", 0, 0, OptionArg.STRING_ARRAY, out arg_files, "Get files", null },
+        { null }
+    };
+
+    public Bluetooth.ObjectManager object_manager;
+    public Bluetooth.Obex.Agent agent_obex;
+    public Bluetooth.Obex.Transfer transfer;
+    public BtResponse bt_response = null;
+    public BtReciever bt_reciever;
+    public BtSender bt_sender;
+    public BtScan bt_scan = null;
+    public GLib.List<BtReciever> bt_recievers;
```

Comment:
Just a nitpicking comment before I test this out:  "Reciever" whereever it occurs should be spelt "Receiver".  "i before e except after c" :wink"

Reply:
ok i will fix it soon.